src/main/java/com/royalrangers/controller/UserController.java

```diff
@@ -51,27 +51,34 @@
     }
 
     @RequestMapping(value = "/user/approve/{id}", method = RequestMethod.GET)
-    public ResponseEntity getUserToApprove(@PathVariable("id") Long platoonId){
+    public ResponseResult getUserToApprove(@PathVariable("id") Long platoonId){
 
         Gson gson  = new Gson();
         String jsonList = gson.toJson(userService.getUsersForApprove(platoonId));
 
-        return new ResponseEntity(ResponseBuilder.success(jsonList), HttpStatus.OK);
+        return ResponseBuilder.success(jsonList);
     }
 
     @RequestMapping(value = "/user/approve/", method = RequestMethod.POST)
-    public ResponseEntity approveUser(@RequestBody List<Long> ids) {
+    public ResponseResult approveUser(@RequestBody List<Long> ids) {
 
         userService.approveUsers(ids);
 
-        return new ResponseEntity(ResponseBuilder.success("Users approved successfully."), HttpStatus.OK);
+        return ResponseBuilder.success("Users approved successfully.");
     }
 
     @RequestMapping(value = "/user/reject/", method = RequestMethod.POST)
-    public ResponseEntity rejectUser(@RequestBody List<Long> ids) {
+    public ResponseResult rejectUser(@RequestBody List<Long> ids) {
 
         userService.rejectUsers(ids);
 
-        return new ResponseEntity(ResponseBuilder.success("Users disabled."), HttpStatus.OK);
+        return ResponseBuilder.success("Users disabled.");
     }
+
+    @GetMapping(value = "/user/rank")
+    public @ResponseBody ResponseResult getUserRankList() {
+        List<Enum> rankList = Arrays.asList(UserRank.values());
```

Comment:
Thats not a good idea to return explicit enum values. Please provide some enum representation values for UI.